C/Clipper/build_tarballs.jl

```diff
@@ -0,0 +1,35 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "Clipper"
+version = v"1.0.0"
+
+# Collection of sources required to build Clipper
+sources = [
+    GitSource("https://github.com/SimonDanisch/ClipperBuilder.git",
+              "9ea1878235d518d5e14bce35a94924a34bab8b68"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/ClipperBuilder/
+mkdir "${libdir}"
+${CXX} -fPIC -std=c++11 -shared -o "${libdir}/libcclipper.${dlext}" clipper.cpp cclipper.cpp
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libcclipper", :cclipper)
```

Comment:
I renamed the product to use the more standard `lib*` prefix

Reply:
I don't see that; but I think the variable most likely should be `libcclipper` as well, although if that's not how Simon wants to organize it, that's fine.  Just a suggestion for uniformity across the ecosystem.

Reply:
I dont really care :D Consistency across the ecosystem seems desirable, though!